content/faq/how-to-publish.md

```diff
@@ -0,0 +1,39 @@
+## How to publish content on LBRY
+
+Advanced users: All of the steps below are for the LBRY app.  For additional details of how to conduct any of these actions through the cli please see https://lbryio.github.io/lbry/cli/
+
+### How to create a channel
+1. Download the LBRY app from lbry.io/get. If you already have the application and it prompts you to update, run the update to be on the most recent version.
+2. Load the application and  click the `Publish` button.
+3. In the `Channel Name` section click the dropdown and select `New Channel` and  declare the name you would like for your channel.  For more details on different channel types, see https://lbry.io/faq/naming
+4. Once your name is selected, there is a `Deposit` section that is below.  It requires a minimum bid of .1 LBC.   See more on deposits here: naming.md.  If you would ever like to abandon your claim there is a guide below in `Abandoning a claim`.
+5. Click `Create Channel` once you have selected your bid amount.  Your deposit is a bid and if this channel isn't claimed by another bid, the channel is yours to use!
+You can revoke this claim if you no longer want to use it with the steps below.
```

Comment:
see the previous comment, abandoning channels is only available via the CLI. Also, it's risky right now because you have to make sure all claims from the channel are abandoned first, and if not, you break your wallet.